##
##
######################################################################################
# THIS PATCH IS MEANT TO APPLY NEEDED CHANGES TO THE NEARCORE PROJECT
# See instructions in the BUILD.md file
######################################################################################
##
##
Index: core/crypto/src/lib.rs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/core/crypto/src/lib.rs b/core/crypto/src/lib.rs
--- a/core/crypto/src/lib.rs	(revision fa305730a0b1fe048c26ff528a0e188a3796df2e)
+++ b/core/crypto/src/lib.rs	(revision c24bd954a7dab65a830306c85353882c8009b831)
@@ -17,7 +17,7 @@
 pub mod key_conversion;
 mod key_file;
 pub mod randomness;
-mod signature;
+pub mod signature;
 mod signer;
 mod test_utils;
 pub mod vrf;


